spec/generators/rspec/controller/controller_generator_spec.rb

```diff
@@ -5,23 +5,29 @@
 RSpec.describe Rspec::Generators::ControllerGenerator, :type => :generator do
   setup_default_destination
 
-  describe 'controller specs' do
-    subject { file('spec/controllers/posts_controller_spec.rb') }
-    describe 'generated by default' do
-      before do
-        run_generator %w(posts)
-      end
+  describe 'request specs' do
+    subject { file('spec/requests/posts_request_spec.rb') }
 
-      describe 'the spec' do
-        it { is_expected.to exist }
-        it { is_expected.to contain(/require 'rails_helper'/) }
-        it { is_expected.to contain(/^RSpec.describe PostsController, #{type_metatag(:controller)}/) }
+    if Rails.version.to_f >= 5.0
```

Comment:
We're dropping official support for versions below 5 so you can exclude this in favour of the else

Reply:
I was guided by [your comment from the issue](https://github.com/rspec/rspec-rails/issues/2056#issuecomment-450561088) 😉

> I'd support a PR that made the behaviour configurable, and changed the default the behaviour of the generator for Rails 5.1, (maybe 5) but older Rails expect controller specs

Anyway, the simpler the better :)